.travis.yml

```diff
@@ -18,7 +18,7 @@ addons:
 
 before_install:
   # Log into GCR
-  - docker login -u _json_key -p "${GOOGLE_JSON_KEY}" https://us.gcr.io
+  - echo "${GOOGLE_JSON_KEY}" | docker login -u _json_key --password-stdin https://us.gcr.io
```

Comment:
This is not the issue.  Travis only provides a password to PRs from a branch on the repo.

You should revert this with:
```
git reset --hard HEAD~1
```

and reopen this PR from a branch on the bazelbuild repo.

Reply:
I got stuck on this for a while and many thanks!!! 

So for external PRs, we need to first review / approve it, then create a PR on the bazelbuild repo?